Tests/ApolloCodegenTests/ApolloCodegenConfigurationCodableTests.swift

```diff
@@ -744,4 +746,57 @@ class ApolloCodegenConfigurationCodableTests: XCTestCase {
       try JSONDecoder().decode(ApolloCodegenConfiguration.APQConfig.self, from: subject)
     ).to(throwError())
   }
+
+  // MARK: - Optional Tests
```

Comment:
I would like to add more tests for optional properties of the JSON decoding - will do in a follow-up PR.